views/backend/login.php

```diff
@@ -27,27 +30,24 @@
                     }else if($model_user->is_admin($email,$password) == 0){
                         $errors['exist']  = "Cet administrateur n'existe pas";
                     }
-
+                    
                     if(!empty($errors)){
                         ?>
                         <div class="card red">
                             <div class="card-content white-text">
                                 <?php
                                     foreach($errors as $error){
-                                        echo $error."<br/>";
+                                        echo $error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5349276)